cabal.project

```diff
@@ -11,8 +11,8 @@ repository cardano-haskell-packages
     d4a35cd3121aa00d18544bb0ac01c3e1691d618f462c46129271bccf39f7e8ee
 
 index-state:
-  , hackage.haskell.org 2024-11-14T09:17:39Z
-  , cardano-haskell-packages 2024-11-20T20:05:41Z
+  , hackage.haskell.org 2025-02-26T10:59:23Z
+  , cardano-haskell-packages 2025-02-15T18:39:38Z
```

Comment:
do we really need this now?

Reply:
Yes, because 1.2.0 is not available with the current indices. Note that indices are not used with plain cabal

Reply:
Ah wait, you mean: do we need pinning down the versions? Perhaps not, indeed

Reply:
ok, next thing: do we really need:
```
active-repositories: hackage.haskell.org, cardano-haskell-packages 
```
?

Reply:
I bet when we will have 1.2.0 in the indices when we would be able to delete `cardano-haskell-packages` and also the definition at the beginning of this file, correct?

Reply:
My reasoning is that we don't publish `cabal.project` to hackage so whatever we put here is only useful within the context of the CI, it doesn't hurt but has to be consistent.